core/sql/regress/compGeneral/EXPECTED042

```diff
@@ -1795,8 +1795,8 @@ HQC key=SELECT T . VARCHAR0_UNIQ AS T_VARCHAR0_UNIQ , T . CHAR2_2 AS T_CHAR2_2 ,
 
 Start Time             2015/08/18 15:48:56.468470
 End Time               2015/08/18 15:48:56.471409
-Elapsed Time                      00:00:00.002939
-Compile Time                      00:00:00.002939
```

Comment:
The compile time is expected to be less than 1 ms.  Last time, when I had
updated the EXPECTED042 file, the compile time in a workstation was more
than 1 ms. So, the expected file was wrong, but the current log file seems
to have less than 1 ms.



Selva



*From:* DaveBirdsall [mailto:notifications@github.com]
*Sent:* Wednesday, September 2, 2015 8:27 AM
*To:* apache/incubator-trafodion <incubator-trafodion@noreply.github.com>
*Subject:* Re: [incubator-trafodion] Fix TRAFODION-1450 (#71)



In core/sql/regress/compGeneral/EXPECTED042
<https://github.com/apache/incubator-trafodion/pull/71#discussion_r38545861>
:

> @@ -1795,8 +1795,8 @@ HQC key=SELECT T . VARCHAR0_UNIQ AS T_VARCHAR0_UNIQ , T . CHAR2_2 AS T_CHAR2_2 ,

>

>  Start Time             2015/08/18 15:48:56.468470

>  End Time               2015/08/18 15:48:56.471409

> -Elapsed Time                      00:00:00.002939

> -Compile Time                      00:00:00.002939

A question to the community: Do we expect these times to match up precisely
on each run or is this something that should be filtered in a test run?

—
Reply to this email directly or view it on GitHub
<https://github.com/apache/incubator-trafodion/pull/71/files#r38545861>.